Added support for clt,r30b1 and build fixes

andros-ua opened the pull request:
Added support for clt,r30b1 in device definition (tested no error on sysupgrade checks).
Minor fixes for build to be done: renamed 7981b.dtsi

andros-ua committed:
Update mt7981b-edup-rt2980.dts

rename "mt7981.dtsi" to "mt7981b.dtsi"

Signed-off-by: andros-ua <46114524+andros-ua@users.noreply.github.com>

andros-ua committed:
Update mt7981b-edup-rt2980n.dts

rename "mt7981.dtsi" to "mt7981b.dtsi"

Signed-off-by: andros-ua <46114524+andros-ua@users.noreply.github.com>

andros-ua committed:
Merge branch 'wb1016:rt2980' into rt2980

andros-ua committed:
Update filogic.mk

Added manufacturer model to supported models of edup rt2980: clt,r30b1

Signed-off-by: andros-ua <46114524+andros-ua@users.noreply.github.com>

andros-ua committed:
Update mt7981b-edup-rt2980.dts

Fix layout according to device default immortalwrt-112m

Signed-off-by: andros-ua <46114524+andros-ua@users.noreply.github.com>

andros-ua committed:
Update mt7981b-edup-rt2980.dts

Changed bootargs

Signed-off-by: andros-ua <46114524+andros-ua@users.noreply.github.com>

Commit:
Fix Network WAN port

wb1016 commented:
nice! i will review this in a day.

Commit:
Fixed assignment of factory MAC addresses.

wb1016 commented on target/linux/mediatek/dts/mt7981b-edup-rt2980.dts:
```diff
@@ -87,72 +87,24 @@
 		compatible = "mediatek,mt7531";
-		reg = <0x1f>;
-		/* Reset GPIO polarity might differ, check board schematics if possible. Assuming ACTIVE_HIGH based on template */
+		reg = <31>;
```
reg 31 thrown dmesg previously.
`[ 1.186526] mdio_bus mdio-bus: switch@1f PHY address 49 is too large`

andros-ua replied:
I don't see error currently:



```
root@OpenWrt:~# dmesg | grep mdio
[    1.006955] mt7530-mdio mdio-bus:1f: configuring for fixed/2500base-x link mode
[    1.015945] mt7530-mdio mdio-bus:1f: Link is Up - 2.5Gbps/Full - flow control rx/tx
[    1.026745] mt7530-mdio mdio-bus:1f lan1 (uninitialized): PHY [mt7530-0:00] driver [MediaTek MT7531 PHY] (irq=79)
[    1.048969] mt7530-mdio mdio-bus:1f lan2 (uninitialized): PHY [mt7530-0:01] driver [MediaTek MT7531 PHY] (irq=80)
[    1.070961] mt7530-mdio mdio-bus:1f lan3 (uninitialized): PHY [mt7530-0:02] driver [MediaTek MT7531 PHY] (irq=81)
[    2.524084] mt7530-mdio mdio-bus:1f lan1: configuring for phy/gmii link mode
[   16.503919] mt7530-mdio mdio-bus:1f lan1: configuring for phy/gmii link mode
[   16.527825] mt7530-mdio mdio-bus:1f lan1: entered allmulticast mode
[   16.547758] mt7530-mdio mdio-bus:1f lan1: entered promiscuous mode
[   16.578854] mt7530-mdio mdio-bus:1f lan2: configuring for phy/gmii link mode
[   16.606297] mt7530-mdio mdio-bus:1f lan2: entered allmulticast mode
[   16.624751] mt7530-mdio mdio-bus:1f lan2: entered promiscuous mode
[   16.647240] mt7530-mdio mdio-bus:1f lan3: configuring for phy/gmii link mode
[   16.669517] mt7530-mdio mdio-bus:1f lan3: entered allmulticast mode
[   16.678899] mt7530-mdio mdio-bus:1f lan3: entered promiscuous mode
[   16.748061] mtk_soc_eth 15100000.ethernet eth1: PHY [mdio-bus:00] driver [MediaTek MT7981 PHY] (irq=POLL)
[  176.636976] mt7530-mdio mdio-bus:1f lan2: Link is Up - 1Gbps/Full - flow control rx/tx
[  343.559983] mt7530-mdio mdio-bus:1f lan2: Link is Down

```

wb1016 replied:
hmm that's strange.

wb1016 commented on target/linux/mediatek/dts/mt7981b-edup-rt2980.dts:
```diff
@@ -250,6 +202,40 @@
+			reg = <2>;
+			label = "lan3";
+		};
+
```
how does this work without the WAN port definition?
port 6 is internal port. port 3 should be the actual WAN.

andros-ua replied:
looks like wan is on SOC itself (eth1) not on the mt7530 lan switch

```
root@OpenWrt:~# dmesg | grep eth
[    0.000000] psci: probing for conduit method from DT.
[    0.830258] mtk_soc_eth 15100000.ethernet: generated random MAC address 65:74:68:25:64:00
[    0.838446] mtk_soc_eth 15100000.ethernet: generated random MAC address 65:74:68:25:64:00
[    0.862126] mtk_soc_eth 15100000.ethernet eth0: mediatek frame engine at 0xffffffc081580000, irq 75
[    0.872014] mtk_soc_eth 15100000.ethernet eth1: mediatek frame engine at 0xffffffc081580000, irq 75
[    1.082747] mtk_soc_eth 15100000.ethernet eth0: entered promiscuous mode
[    2.491650] mtk_soc_eth 15100000.ethernet eth0: configuring for fixed/2500base-x link mode
[    2.500205] mtk_soc_eth 15100000.ethernet eth0: Link is Up - 2.5Gbps/Full - flow control rx/tx
[   16.470406] mtk_soc_eth 15100000.ethernet eth0: Link is Down
[   16.487916] mtk_soc_eth 15100000.ethernet eth0: configuring for fixed/2500base-x link mode
[   16.500445] mtk_soc_eth 15100000.ethernet eth0: Link is Up - 2.5Gbps/Full - flow control rx/tx
[   16.534137] mtk_soc_eth 15100000.ethernet eth0: entered allmulticast mode
[   16.748061] mtk_soc_eth 15100000.ethernet eth1: PHY [mdio-bus:00] driver [MediaTek MT7981 PHY] (irq=POLL)
[   16.757703] mtk_soc_eth 15100000.ethernet eth1: configuring for phy/gmii link mode
[   16.810659] mtk_soc_eth 15100000.ethernet eth1: Link is Up - 1Gbps/Full - flow control rx/tx

```

wb1016 reviewed (commented):
nice!

wb1016 commented:
did you tested this on actual build?

andros-ua commented:
> did you tested this on actual build?

On master and on backported to stable variant:

```
root@OpenWrt:~# dmesg
[    0.000000] Booting Linux on physical CPU 0x0000000000 [0x410fd034]
[    0.000000] Linux version 6.6.92 (andros@openwrt-fork) (aarch64-openwrt-linux-musl-gcc (OpenWrt GCC 13.3.0 r28687-79cfac227e) 13.3.0, GNU ld (GNU Binutils) 2.42) #0 SMP Sun Jun  8 19:25:07 2025
[    0.000000] Machine model: EDUP RT2980
[    0.000000] earlycon: uart8250 at MMIO32 0x0000000011002000 (options '')
[    0.000000] printk: bootconsole [uart8250] enabled
[    0.000000] OF: reserved mem: 0x0000000042ff0000..0x0000000042ffffff (64 KiB) map non-reusable ramoops@42ff0000
[    0.000000] OF: reserved mem: 0x0000000043000000..0x000000004302ffff (192 KiB) nomap non-reusable secmon@43000000
[    0.000000] OF: reserved mem: 0x0000000047c80000..0x0000000047d7ffff (1024 KiB) nomap non-reusable wmcpu-reserved@47c80000
[    0.000000] OF: reserved mem: 0x0000000047d80000..0x0000000047dbffff (256 KiB) nomap non-reusable wo-emi@47d80000
[    0.000000] OF: reserved mem: 0x0000000047dc0000..0x0000000047ffffff (2304 KiB) nomap non-reusable wo-data@47dc0000
[    0.000000] Zone ranges:
[    0.000000]   DMA      [mem 0x0000000040000000-0x000000004fffffff]
[    0.000000]   DMA32    empty
[    0.000000]   Normal   empty
[    0.000000] Movable zone start for each node
[    0.000000] Early memory node ranges
[    0.000000]   node   0: [mem 0x0000000040000000-0x0000000042ffffff]
[    0.000000]   node   0: [mem 0x0000000043000000-0x000000004302ffff]
[    0.000000]   node   0: [mem 0x0000000043030000-0x0000000047c7ffff]
[    0.000000]   node   0: [mem 0x0000000047c80000-0x0000000047ffffff]
[    0.000000]   node   0: [mem 0x0000000048000000-0x000000004fffffff]
[    0.000000] Initmem setup node 0 [mem 0x0000000040000000-0x000000004fffffff]
[    0.000000] psci: probing for conduit method from DT.
[    0.000000] psci: PSCIv1.1 detected in firmware.
[    0.000000] psci: Using standard PSCI v0.2 function IDs
[    0.000000] psci: MIGRATE_INFO_TYPE not supported.
[    0.000000] psci: SMC Calling Convention v1.2
[    0.000000] percpu: Embedded 18 pages/cpu s35048 r8192 d30488 u73728
[    0.000000] pcpu-alloc: s35048 r8192 d30488 u73728 alloc=18*4096
[    0.000000] pcpu-alloc: [0] 0 [0] 1
[    0.000000] Detected VIPT I-cache on CPU0
[    0.000000] CPU features: detected: GIC system register CPU interface
[    0.000000] CPU features: kernel page table isolation disabled by kernel configuration
[    0.000000] alternatives: applying boot alternatives
[    0.000000] Kernel command line: console=ttyS0,115200n1 loglevel=8 earlycon=uart8250,mmio32,0x11002000 ubi.mtd=ubi
[    0.000000] Dentry cache hash table entries: 32768 (order: 6, 262144 bytes, linear)
[    0.000000] Inode-cache hash table entries: 16384 (order: 5, 131072 bytes, linear)
[    0.000000] Built 1 zonelists, mobility grouping on.  Total pages: 64512
[    0.000000] mem auto-init: stack:off, heap alloc:off, heap free:off
[    0.000000] software IO TLB: SWIOTLB bounce buffer size adjusted to 0MB
[    0.000000] software IO TLB: area num 2.
[    0.000000] software IO TLB: SWIOTLB bounce buffer size roundup to 0MB
[    0.000000] software IO TLB: mapped [mem 0x000000004fe48000-0x000000004fec8000] (0MB)
[    0.000000] Memory: 239184K/262144K available (8832K kernel code, 994K rwdata, 2568K rodata, 448K init, 289K bss, 22960K reserved, 0K cma-reserved)
[    0.000000] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
[    0.000000] rcu: Hierarchical RCU implementation.
[    0.000000] rcu:     RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
[    0.000000]  Tracing variant of Tasks RCU enabled.
[    0.000000] rcu: RCU calculated value of scheduler-enlistment delay is 10 jiffies.
[    0.000000] rcu: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=2
[    0.000000] NR_IRQS: 64, nr_irqs: 64, preallocated irqs: 0
[    0.000000] GICv3: GIC: Using split EOI/Deactivate mode
[    0.000000] GICv3: 640 SPIs implemented
[    0.000000] GICv3: 0 Extended SPIs implemented
[    0.000000] Root IRQ handler: gic_handle_irq
[    0.000000] GICv3: GICv3 features: 16 PPIs
[    0.000000] GICv3: CPU0: found redistributor 0 region 0:0x000000000c080000
[    0.000000] rcu: srcu_init: Setting srcu_struct sizes based on contention.
[    0.000000] arch_timer: cp15 timer(s) running at 13.00MHz (phys).
[    0.000000] clocksource: arch_sys_counter: mask: 0xffffffffffffff max_cycles: 0x2ff89eacb, max_idle_ns: 440795202429 ns
[    0.000000] sched_clock: 56 bits at 13MHz, resolution 76ns, wraps every 4398046511101ns
[    0.008128] Calibrating delay loop (skipped), value calculated using timer frequency.. 26.00 BogoMIPS (lpj=130000)
[    0.018537] pid_max: default: 32768 minimum: 301
[    0.026067] Mount-cache hash table entries: 512 (order: 0, 4096 bytes, linear)
[    0.033333] Mountpoint-cache hash table entries: 512 (order: 0, 4096 bytes, linear)
[    0.043144] cacheinfo: Unable to detect cache hierarchy for CPU 0
[    0.049821] RCU Tasks Trace: Setting shift to 1 and lim to 1 rcu_task_cb_adjust=1 rcu_task_cpu_ids=2.
[    0.059236] rcu: Hierarchical SRCU implementation.
[    0.064064] rcu:     Max phase no-delay instances is 1000.
[    0.069703] smp: Bringing up secondary CPUs ...
[    0.074651] Detected VIPT I-cache on CPU1
[    0.074695] GICv3: CPU1: found redistributor 1 region 0:0x000000000c0a0000
[    0.074723] CPU1: Booted secondary processor 0x0000000001 [0x410fd034]
[    0.074790] smp: Brought up 1 node, 2 CPUs
[    0.096401] SMP: Total of 2 processors activated.
[    0.101124] CPU features: detected: 32-bit EL0 Support
[    0.106282] CPU features: detected: CRC32 instructions
[    0.111474] CPU features: emulated: Privileged Access Never (PAN) using TTBR0_EL1 switching
[    0.119865] CPU: All CPU(s) started at EL2
[    0.123978] alternatives: applying system-wide alternatives
[    0.133026] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
[    0.142935] futex hash table entries: 512 (order: 3, 32768 bytes, linear)
[    0.151016] pinctrl core: initialized pinctrl subsystem
[    0.157420] NET: Registered PF_NETLINK/PF_ROUTE protocol family
[    0.163707] DMA: preallocated 128 KiB GFP_KERNEL pool for atomic allocations
[    0.170834] DMA: preallocated 128 KiB GFP_KERNEL|GFP_DMA pool for atomic allocations
[    0.178631] DMA: preallocated 128 KiB GFP_KERNEL|GFP_DMA32 pool for atomic allocations
[    0.186964] thermal_sys: Registered thermal governor 'fair_share'
[    0.186970] thermal_sys: Registered thermal governor 'bang_bang'
[    0.193089] thermal_sys: Registered thermal governor 'step_wise'
[    0.199129] thermal_sys: Registered thermal governor 'user_space'
[    0.205241] ASID allocator initialised with 65536 entries
[    0.217497] pstore: Using crash dump compression: deflate
[    0.222934] pstore: Registered ramoops as persistent store backend
[    0.229140] ramoops: using 0x10000@0x42ff0000, ecc: 0
[    0.235606] /soc/interrupt-controller@c000000: Fixed dependency cycle(s) with /soc/interrupt-controller@c000000
[    0.251048] Modules: 29440 pages in range for non-PLT usage
[    0.251058] Modules: 520960 pages in range for PLT usage
[    0.257793] cryptd: max_cpu_qlen set to 1000
[    0.269550] SCSI subsystem initialized
[    0.273498] libata version 3.00 loaded.
[    0.278894] clocksource: Switched to clocksource arch_sys_counter
[    0.287316] NET: Registered PF_INET protocol family
[    0.292350] IP idents hash table entries: 4096 (order: 3, 32768 bytes, linear)
[    0.300729] tcp_listen_portaddr_hash hash table entries: 256 (order: 0, 4096 bytes, linear)
[    0.309145] Table-perturb hash table entries: 65536 (order: 6, 262144 bytes, linear)
[    0.316931] TCP established hash table entries: 2048 (order: 2, 16384 bytes, linear)
[    0.324730] TCP bind hash table entries: 2048 (order: 4, 65536 bytes, linear)
[    0.331950] TCP: Hash tables configured (established 2048 bind 2048)
[    0.338646] MPTCP token hash table entries: 256 (order: 0, 6144 bytes, linear)
[    0.346032] UDP hash table entries: 256 (order: 1, 8192 bytes, linear)
[    0.352609] UDP-Lite hash table entries: 256 (order: 1, 8192 bytes, linear)
[    0.359812] NET: Registered PF_UNIX/PF_LOCAL protocol family
[    0.365524] PCI: CLS 0 bytes, default 64
[    0.370651] workingset: timestamp_bits=46 max_order=16 bucket_order=0
[    0.381899] squashfs: version 4.0 (2009/01/31) Phillip Lougher
[    0.387767] jffs2: version 2.2 (NAND) (SUMMARY) (LZMA) (RTIME) (CMODE_PRIORITY) (c) 2001-2006 Red Hat, Inc.
[    0.464060] Serial: 8250/16550 driver, 3 ports, IRQ sharing disabled
[    0.471704] printk: console [ttyS0] disabled
[    0.496367] 11002000.serial: ttyS0 at MMIO 0x11002000 (irq = 72, base_baud = 2500000) is a ST16650V2
[    0.505595] printk: console [ttyS0] enabled
[    0.513962] printk: bootconsole [uart8250] disabled
[    0.528339] loop: module loaded
[    0.534484] spi-nand spi0.0: calibration result: 0x3
[    0.539559] spi-nand spi0.0: ESMT SPI NAND was found.
[    0.544603] spi-nand spi0.0: 128 MiB, block size: 128 KiB, page size: 2048, OOB size: 64
[    0.553112] Signature found at block 1023 [0x07fe0000]
[    0.558244] NMBM management region starts at block 960 [0x07800000]
[    0.565546] First info table with writecount 0 found in block 960
[    0.574465] Second info table with writecount 0 found in block 963
[    0.580649] NMBM has been successfully attached
[    0.585484] 6 fixed-partitions partitions found on MTD device spi0.0
[    0.591851] Creating 6 MTD partitions on "spi0.0":
[    0.596632] 0x000000000000-0x000008000000 : "spi0.0"
[    0.601589] mtd: partition "spi0.0" extends beyond the end of device "spi0.0" -- size truncated to 0x7800000
[    0.668924] 0x000000000000-0x000000100000 : "BL2"
[    0.674019] 0x000000100000-0x000000180000 : "u-boot-env"
[    0.679777] 0x000000180000-0x000000380000 : "Factory"
[    0.685597] 0x000000380000-0x000000580000 : "FIP"
[    0.690753] 0x000000580000-0x000007580000 : "ubi"
[    0.830178] mtk_soc_eth 15100000.ethernet: generated random MAC address 65:74:68:25:64:00
[    0.838366] mtk_soc_eth 15100000.ethernet: generated random MAC address 65:74:68:25:64:00
[    0.862062] mtk_soc_eth 15100000.ethernet eth0: mediatek frame engine at 0xffffffc081500000, irq 75
[    0.871978] mtk_soc_eth 15100000.ethernet eth1: mediatek frame engine at 0xffffffc081500000, irq 75
[    0.881674] i2c_dev: i2c /dev entries driver
[    0.887647] mtk-wdt 1001c000.watchdog: Watchdog enabled (timeout=31 sec, nowayout=0)
[    0.896735] NET: Registered PF_INET6 protocol family
[    0.902791] Segment Routing with IPv6
[    0.906476] In-situ OAM (IOAM) with IPv6
[    0.910477] NET: Registered PF_PACKET protocol family
[    0.915655] 8021q: 802.1Q VLAN Support v1.8
[    1.008212] mt7530-mdio mdio-bus:1f: configuring for fixed/2500base-x link mode
[    1.017236] mt7530-mdio mdio-bus:1f: Link is Up - 2.5Gbps/Full - flow control rx/tx
[    1.027977] mt7530-mdio mdio-bus:1f lan1 (uninitialized): PHY [mt7530-0:00] driver [MediaTek MT7531 PHY] (irq=79)
[    1.050225] mt7530-mdio mdio-bus:1f lan2 (uninitialized): PHY [mt7530-0:01] driver [MediaTek MT7531 PHY] (irq=80)
[    1.072219] mt7530-mdio mdio-bus:1f lan3 (uninitialized): PHY [mt7530-0:02] driver [MediaTek MT7531 PHY] (irq=81)
[    1.084033] mtk_soc_eth 15100000.ethernet eth0: entered promiscuous mode
[    1.090793] DSA: tree 0 setup
[    1.094106] ubi0: default fastmap pool size: 40
[    1.098637] ubi0: default fastmap WL pool size: 20
[    1.103470] ubi0: attaching mtd5
[    1.490321] ubi0: scanning is finished
[    1.500300] ubi0: attached mtd5 (name "ubi", size 112 MiB)
[    1.505789] ubi0: PEB size: 131072 bytes (128 KiB), LEB size: 126976 bytes
[    1.512664] ubi0: min./max. I/O unit sizes: 2048/2048, sub-page size 2048
[    1.519446] ubi0: VID header offset: 2048 (aligned 2048), data offset: 4096
[    1.526394] ubi0: good PEBs: 896, bad PEBs: 0, corrupted PEBs: 0
[    1.532391] ubi0: user volume: 3, internal volumes: 1, max. volumes count: 128
[    1.539602] ubi0: max/mean erase counter: 4/2, WL threshold: 4096, image sequence number: 1058065918
[    1.548718] ubi0: available PEBs: 0, total reserved PEBs: 896, PEBs reserved for bad PEB handling: 19
[    1.557931] ubi0: background thread "ubi_bgt0d" started, PID 544
[    1.558621] block ubiblock0_1: created from ubi0:1(rootfs)
[    1.569432] ubiblock: device ubiblock0_1 (rootfs) set to be root filesystem
[    1.576483] clk: Disabling unused clocks
[    1.585644] VFS: Mounted root (squashfs filesystem) readonly on device 254:0.
[    1.592997] Freeing unused kernel memory: 448K
[    1.597513] Run /sbin/init as init process
[    1.601613]   with arguments:
[    1.604567]     /sbin/init
[    1.607262]   with environment:
[    1.610399]     HOME=/
[    1.612747]     TERM=linux
[    1.808216] init: Console is alive
[    1.811814] init: - watchdog -
[    2.182320] kmodloader: loading kernel modules from /etc/modules-boot.d/*
[    2.209474] gpio_button_hotplug: loading out-of-tree module taints kernel.
[    2.219158] kmodloader: done loading kernel modules from /etc/modules-boot.d/*
[    2.229278] init: - preinit -
[    2.318916] random: crng init done
[    2.563859] mtk_soc_eth 15100000.ethernet eth0: configuring for fixed/2500base-x link mode
[    2.572377] mtk_soc_eth 15100000.ethernet eth0: Link is Up - 2.5Gbps/Full - flow control rx/tx
[    2.595531] mt7530-mdio mdio-bus:1f lan1: configuring for phy/gmii link mode
[    6.787408] UBIFS (ubi0:2): default file-system created
[    6.793111] UBIFS (ubi0:2): Mounting in unauthenticated mode
[    6.798877] UBIFS (ubi0:2): background thread "ubifs_bgt0_2" started, PID 663
[    6.840806] UBIFS (ubi0:2): UBIFS: mounted UBI device 0, volume 2, name "rootfs_data"
[    6.848634] UBIFS (ubi0:2): LEB size: 126976 bytes (124 KiB), min./max. I/O unit sizes: 2048 bytes/2048 bytes
[    6.858564] UBIFS (ubi0:2): FS size: 99803136 bytes (95 MiB, 786 LEBs), max 797 LEBs, journal size 4952064 bytes (4 MiB, 39 LEBs)
[    6.870206] UBIFS (ubi0:2): reserved for root: 4713948 bytes (4603 KiB)
[    6.876807] UBIFS (ubi0:2): media format: w5/r0 (latest is w5/r0), UUID F2791692-3F8E-4701-B868-78C5B9E5F08C, small LPT model
[    6.889572] mount_root: overlay filesystem has not been fully initialized yet
[    6.896922] mount_root: switching to ubifs overlay
[    6.904465] overlayfs: null uuid detected in lower fs '/', falling back to xino=off,index=off,nfs_export=off.
[    6.918326] urandom-seed: Seed file not found (/etc/urandom.seed)
[    6.966450] procd: - early -
[    6.969444] procd: - watchdog -
[    7.513375] procd: - watchdog -
[    7.566024] procd: - ubus -
[    7.722291] procd: - init -
[    8.050616] kmodloader: loading kernel modules from /etc/modules.d/*
[    8.088365] crypto-safexcel 10320000.crypto: EIP97:230(0,1,4,4)-HIA:270(0,5,5),PE:150/433(alg:7fcdfc00)/0/0/0
[    8.115953] Loading modules backported from Linux version v6.12.6-0-ge9d65b48ce1a
[    8.123463] Backport generated by backports.git v6.1.110-1-35-g410656ef04d2
[    8.262105] urngd: v1.0.2 started.
[    8.524241] mt798x-wmac 18000000.wifi: HW/SW Version: 0x8a108a10, Build Time: 20240823161240a
[    8.524241]
[    8.726160] mt798x-wmac 18000000.wifi: WM Firmware Version: ____000000, Build Time: 20240823161304
[    8.820277] mt798x-wmac 18000000.wifi: WA Firmware Version: DEV_000000, Build Time: 20240823161841
[    8.918001] mt798x-wmac 18000000.wifi: registering led 'mt76-phy0'
[    8.960917] mt798x-wmac 18000000.wifi: registering led 'mt76-phy1'
[    9.025771] PPP generic driver version 2.4.2
[    9.031316] NET: Registered PF_PPPOX protocol family
[    9.040060] kmodloader: done loading kernel modules from /etc/modules.d/*
[   10.264691] mtdblock: MTD device 'Factory' is NAND, please consider using UBI block devices instead.
[   10.289297] mtdblock: MTD device 'Factory' is NAND, please consider using UBI block devices instead.
[   10.313938] mtdblock: MTD device 'Factory' is NAND, please consider using UBI block devices instead.
[   15.902125] mtk_soc_eth 15100000.ethernet eth0: Link is Down
[   15.924085] mtk_soc_eth 15100000.ethernet eth0: configuring for fixed/2500base-x link mode
[   15.932811] mtk_soc_eth 15100000.ethernet eth0: Link is Up - 2.5Gbps/Full - flow control rx/tx
[   15.940714] mt7530-mdio mdio-bus:1f lan1: configuring for phy/gmii link mode
[   15.953283] br-lan: port 1(lan1) entered blocking state
[   15.958515] br-lan: port 1(lan1) entered disabled state
[   15.963843] mt7530-mdio mdio-bus:1f lan1: entered allmulticast mode
[   15.970147] mtk_soc_eth 15100000.ethernet eth0: entered allmulticast mode
[   15.980148] mt7530-mdio mdio-bus:1f lan1: entered promiscuous mode
[   16.028745] mt7530-mdio mdio-bus:1f lan2: configuring for phy/gmii link mode
[   16.040761] br-lan: port 2(lan2) entered blocking state
[   16.045996] br-lan: port 2(lan2) entered disabled state
[   16.051317] mt7530-mdio mdio-bus:1f lan2: entered allmulticast mode
[   16.060024] mt7530-mdio mdio-bus:1f lan2: entered promiscuous mode
[   16.076118] mt7530-mdio mdio-bus:1f lan3: configuring for phy/gmii link mode
[   16.087386] br-lan: port 3(lan3) entered blocking state
[   16.092682] br-lan: port 3(lan3) entered disabled state
[   16.097937] mt7530-mdio mdio-bus:1f lan3: entered allmulticast mode
[   16.106213] mt7530-mdio mdio-bus:1f lan3: entered promiscuous mode
[   16.144325] mtk_soc_eth 15100000.ethernet eth1: PHY [mdio-bus:00] driver [MediaTek MT7981 PHY] (irq=POLL)
[   16.153962] mtk_soc_eth 15100000.ethernet eth1: configuring for phy/gmii link mode
[   16.184680] mtk_soc_eth 15100000.ethernet eth1: Link is Up - 1Gbps/Full - flow control rx/tx
[   39.639582] mt7530-mdio mdio-bus:1f lan2: Link is Up - 1Gbps/Full - flow control rx/tx
[   39.639608] br-lan: port 2(lan2) entered blocking state
[   39.652719] br-lan: port 2(lan2) entered forwarding state
[  135.457107] br-lan: port 2(lan2) entered disabled state
[  135.457223] mt7530-mdio mdio-bus:1f lan2: Link is Down
[  269.611154] br-lan: port 4(phy1-ap0) entered blocking state
[  269.616787] br-lan: port 4(phy1-ap0) entered disabled state
[  269.622387] mt798x-wmac 18000000.wifi phy1-ap0: entered allmulticast mode
[  269.629405] mt798x-wmac 18000000.wifi phy1-ap0: entered promiscuous mode
[  351.968862] br-lan: port 4(phy1-ap0) entered blocking state
[  351.974451] br-lan: port 4(phy1-ap0) entered forwarding state

```
